micrometer-core/src/main/java/io/micrometer/core/instrument/distribution/TimeWindowFixedBoundaryHistogram.java

```diff
@@ -38,28 +38,65 @@ public class TimeWindowFixedBoundaryHistogram extends AbstractTimeWindowHistogra
 
     private final boolean isCumulativeBucketCounts;
 
+    /**
+     * Create a {@code TimeWindowFixedBoundaryHistogram} instance with cumulative bucket
+     * counts and buckets based on the {@link DistributionStatisticConfig config} and
+     * {@code supportsAggregablePercentiles}.
+     * @param clock clock
+     * @param config distribution statistic configuration
+     * @param supportsAggregablePercentiles whether the backend supports aggregable
+     * percentiles
+     * @see PercentileHistogramBuckets
+     */
     public TimeWindowFixedBoundaryHistogram(Clock clock, DistributionStatisticConfig config,
             boolean supportsAggregablePercentiles) {
         this(clock, config, supportsAggregablePercentiles, true);
     }
 
     /**
-     * Create a {@code TimeWindowFixedBoundaryHistogram} instance.
+     * Create a {@code TimeWindowFixedBoundaryHistogram} instance with buckets based on
```

Comment:
Should we mention here and in the docs of the previous ctor what are the defaults?
Here: inf bucket is not forced/might be missing.

Reply:
I was trying to allude to that by saying the buckets are based on the DistributionStatisticConfig and supportsAggregablePercentiles. If those result in an inf bucket, it'll be there. If not, it won't.